### PR TITLE
Fix a test

### PR DESCRIPTION
**Pull request by mulkieran**

### CODE REVIEW

File: tests/whitebox/integration/test_stratis.py [hunk 1]
```diff
@@ -92,11 +92,20 @@ class ErrorHandlingTestCase(SimTestCase):
...
+        exit.
         """
-        command_line = ["pool", "list", "not_existing"]
+        command_line = ["filesystem", "list", "not_existing"]
+
+        # if exceptions are propagated then a Stratis error is caught
+        with self.assertRaises(StratisCliActionError):
+            RUNNER(["--propagate"] + command_line)
+
+        # If instead the exception chain is handed of to handle_error,
```
Comment (bgurney-rh) [inline tests/whitebox/integration/test_stratis.py]: "If instead the exception chain is handed off to handle_error,"?

Comment (mulkieran) [reply tests/whitebox/integration/test_stratis.py]: ```handle_error``` is the method that tries to understand the exception chain if the exception is not just propagated.

Comment (jbaublitz) [reply tests/whitebox/integration/test_stratis.py]: I think it's more a question of if it's off or of. I was wondering if that's a typo too.

Comment (mulkieran) [reply tests/whitebox/integration/test_stratis.py]: Oh! I'll fix that typo.

**Comment by mulkieran** [comment]
typo fixed.

**Pull request merged**